modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/upgrade/v7_0_0/test/CommerceAddressUpgradeProcessTest.java

```diff
@@ -0,0 +1,154 @@
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.upgrade.registry.UpgradeStepRegistrator;
+import com.liferay.portal.upgrade.test.util.UpgradeTestUtil;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Brian I. Kim
+ */
+@RunWith(Arquillian.class)
+public class CommerceAddressUpgradeProcessTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	@Test
+	public void testUpdateCommerceAddress() throws Exception {
+		_company = CompanyTestUtil.addCompany();
+
+		CommerceAddress commerceAddress = _addCommerceAddress(
+			_company.getCompanyId(), _company.getGroupId());
+
+		_runUpgrade();
+
+		EntityCacheUtil.clearCache();
+
+		Address address =
+			_addressLocalService.fetchAddressByExternalReferenceCode(
+				String.valueOf(commerceAddress.getCommerceAddressId()),
+				_company.getCompanyId());
+
+		Assert.assertNotNull(address);
+
+		Assert.assertEquals(
+			commerceAddress.getClassNameId(), address.getClassNameId());
+		Assert.assertEquals(commerceAddress.getClassPK(), address.getClassPK());
+		Assert.assertEquals(
+			commerceAddress.getCompanyId(), address.getCompanyId());
+		Assert.assertEquals(commerceAddress.getName(), address.getName());
+		Assert.assertEquals(commerceAddress.getUserId(), address.getUserId());
+
+		ListType listType = _listTypeLocalService.getListType(
+			_company.getCompanyId(),
+			AccountListTypeConstants.
+				ACCOUNT_ENTRY_ADDRESS_TYPE_BILLING_AND_SHIPPING,
+			AccountListTypeConstants.ACCOUNT_ENTRY_ADDRESS);
+
+		Assert.assertEquals(listType.getListTypeId(), address.getListTypeId());
+	}
+
+	private CommerceAddress _addCommerceAddress(long companyId, long groupId)
+		throws Exception {
+
+		Country country = _countryLocalService.fetchCountryByA2(
+			companyId, "US");
+
+		Region region = _regionLocalService.getRegion(
+			country.getCountryId(), "CA");
+
+		User user = UserTestUtil.getAdminUser(_company.getCompanyId());
+
+		AccountEntry accountEntry =
+			CommerceAccountTestUtil.addBusinessAccountEntry(
+				user.getUserId(), "Test Business Account", null, null,
+				new long[] {user.getUserId()}, null,
+				ServiceContextTestUtil.getServiceContext(groupId));
+
+		return _commerceAddressLocalService.addCommerceAddress(
+			StringPool.BLANK, AccountEntry.class.getName(),
+			accountEntry.getAccountEntryId(), country.getCountryId(),
+			region.getRegionId(), RandomTestUtil.randomString(),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+			StringPool.BLANK,
+			CommerceAddressConstants.ADDRESS_TYPE_BILLING_AND_SHIPPING,
+			RandomTestUtil.randomString(),
+			ServiceContextTestUtil.getServiceContext(groupId));
```

Comment:
I think this test is wrong because now, as far as I know, if we use the _commerceAddressLocalService.addCommerceAddress it already saves the data in the Address table and not in the CommerceAddress one so the upgrade process doesn't "migrate" nothing